pkgs/development/python-modules/json-logging/default.nix

```diff
@@ -29,11 +30,12 @@ buildPythonPackage rec {
     })
   ];
 
-  # - Quart is not packaged for Nixpkgs.
-  # - FastAPI is broken, see #112701 and tiangolo/fastapi#2335.
-  checkInputs = [ wheel flask /*quart*/ sanic /*fastapi*/ uvicorn requests pytestCheckHook ];
-  disabledTests = [ "quart" "fastapi" ];
-  disabledTestPaths = [ "tests/test_fastapi.py" ];
+  checkInputs = [ wheel flask /*quart*/ sanic fastapi uvicorn requests pytestCheckHook ];
+  disabledTests = [
+    "quart" # Quart is not packaged for Nixpkgs
+  ] ++ lib.optionals (pythonAtLeast "3.9") [
+    "test_api_example" # network access
```

Comment:
This is really strange.

Reply:
Did urllib3 somehow strange in Python 3.9 in a way such that local networking doesn't work in the sandbox anymore?
```
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPConnection object at 0x7ffff50a5f70>: Failed to establish a new connection: [Errno 111] Connection refused
```
cc @FRidh @jonringer